composer.json

```diff
@@ -20,7 +20,7 @@
                 "version": "0.3.0-rc6",
                 "type": "drupal-library",
                 "dist": {
-                    "url": "https://github.com/reload/dpl-react/releases/download/release-indloggetudlogget-funktionalitet-DDFSOEG-178/dist.zip",
+                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-react/releases/download/release-release%2F4/dist.zip",
```

Comment:
:+1: It would be nice if we could have some kind of warning to prevent merges of local builds to main branches.

Reply:
yes 🎄